cinema/models.py

```diff
@@ -35,12 +39,21 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def create_custom_path(instance, filename):
```

Comment:
Since you have hardcoded `"uploads/movies/"` it's not just `create_custom_path` it's more like `create_movie_image_path`